KrakenIoc/KrakenIoc.Testing/V1/InjectorTests.cs

```diff
@@ -80,25 +92,51 @@ public void Injector_DoesNotThrowException_WhenMultipleConstructors()
         }
 
```

Comment:
I'd like to see test coverage to ensure the following cases are successful as well:
```
BaseType: public get;
DerivedType: private set;
```

Reply:
This has been covered.